packages/stylelint-config/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@readme/stylelint-config",
+  "version": "1.0.0",
+  "description": "ReadMe coding standards for styles",
+  "main": "index.js",
+  "files": [
+    "index.js"
+  ],
+  "scripts": {
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/readmeio/stylelint-config.git"
+  },
+  "author": "Engineers <engineers@readme.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/readmeio/stylelint-config/issues"
+  },
+  "homepage": "https://github.com/readmeio/stylelint-config#readme",
+  "dependencies": {
+    "postcss": "^8.4.12",
+    "stylelint": "^14.6.0",
```

Comment:
Might want to make this a peerDep.

Reply:
Yes, will do. I think I installed this and the postcss deps when initially doing some tests with the anticipation these will need to be cleaned up before merging.

Reply:
Done